ocaml/tests/test_vm_helpers.ml

```diff
@@ -70,7 +70,7 @@ let test_gpus_available_fails_no_pgpu () =
       let group = List.hd (Db.GPU_group.get_all ~__context) in
       let vm = make_vm_with_vgpu_in_group ~__context VGPU_T.k100 group in
       assert_raises_api_error
-        Api_errors.vm_requires_gpu
+        Api_errors.vm_requires_vgpu
```

Comment:
You may feel confused by this.
A vm_requires_gpu error was expected here. But by the design of the new function, it should be vm_requires_vgpu here.